The failures look legitimate:

```
FAILED: bin/ze_validation_layer.dll lib/ze_validation_layer.lib 
C:\Windows\system32\cmd.exe /C "cd . && D:\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=source\layers\validation\CMakeFiles\ze_validation_layer.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\link.exe  source\layers\validation\CMakeFiles\ze_validation_layer.dir\ze_validation_layer.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\ze_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\zet_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\zes_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\ZeValidationLayerVersion.rc.res source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\ze_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\zes_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\zet_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\basic_leak\zel_basic_leak_checker.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\events_checker\zel_events_checker.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\__\__\__\third_party\xla\graphcycles.cc.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\ze_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\zes_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\zet_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\template\zel_template_checker.cpp.obj  /out:bin\ze_validation_layer.dll /implib:lib\ze_validation_layer.lib /pdb:bin\ze_validation_layer.pdb /dll /version:1.20 /machine:x64 /DYNAMICBASE /guard:cf /CETCOMPAT /nologo    /debug /INCREMENTAL  lib\level_zero_utils.lib  D:\installed\x64-windows\debug\lib\spdlogd.lib  D:\installed\x64-windows\debug\lib\fmtd.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK Pass 1: command "C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\link.exe source\layers\validation\CMakeFiles\ze_validation_layer.dir\ze_validation_layer.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\ze_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\zet_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\zes_valddi.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\ZeValidationLayerVersion.rc.res source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\ze_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\zes_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\handle_lifetime_tracking\zet_handle_lifetime.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\basic_leak\zel_basic_leak_checker.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\events_checker\zel_events_checker.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\__\__\__\third_party\xla\graphcycles.cc.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\ze_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\zes_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\parameter_validation\zet_parameter_validation.cpp.obj source\layers\validation\CMakeFiles\ze_validation_layer.dir\checkers\template\zel_template_checker.cpp.obj /out:bin\ze_validation_layer.dll /implib:lib\ze_validation_layer.lib /pdb:bin\ze_validation_layer.pdb /dll /version:1.20 /machine:x64 /DYNAMICBASE /guard:cf /CETCOMPAT /nologo /debug /INCREMENTAL lib\level_zero_utils.lib D:\installed\x64-windows\debug\lib\spdlogd.lib D:\installed\x64-windows\debug\lib\fmtd.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:source\layers\validation\CMakeFiles\ze_validation_layer.dir/intermediate.manifest source\layers\validation\CMakeFiles\ze_validation_layer.dir/manifest.res" failed (exit code 1120) with the following output:
   Creating library lib\ze_validation_layer.lib and object lib\ze_validation_layer.exp
ze_validation_layer.cpp.obj : error LNK2019: unresolved external symbol "public: __cdecl spdlog::sinks::ansicolor_stdout_sink<struct spdlog::details::console_mutex>::ansicolor_stdout_sink<struct spdlog::details::console_mutex>(enum spdlog::color_mode)" (??0?$ansicolor_stdout_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@QEAA@W4color_mode@2@@Z) referenced in function "void __cdecl std::_Construct_in_place<class spdlog::sinks::ansicolor_stdout_sink<struct spdlog::details::console_mutex> >(class spdlog::sinks::ansicolor_stdout_sink<struct spdlog::details::console_mutex> &)" (??$_Construct_in_place@V?$ansicolor_stdout_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@$$V@std@@YAXAEAV?$ansicolor_stdout_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@@Z)
ze_validation_layer.cpp.obj : error LNK2019: unresolved external symbol "public: __cdecl spdlog::sinks::ansicolor_stderr_sink<struct spdlog::details::console_mutex>::ansicolor_stderr_sink<struct spdlog::details::console_mutex>(enum spdlog::color_mode)" (??0?$ansicolor_stderr_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@QEAA@W4color_mode@2@@Z) referenced in function "void __cdecl std::_Construct_in_place<class spdlog::sinks::ansicolor_stderr_sink<struct spdlog::details::console_mutex> >(class spdlog::sinks::ansicolor_stderr_sink<struct spdlog::details::console_mutex> &)" (??$_Construct_in_place@V?$ansicolor_stderr_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@$$V@std@@YAXAEAV?$ansicolor_stderr_sink@Uconsole_mutex@details@spdlog@@@sinks@spdlog@@@Z)
bin\ze_validation_layer.dll : fatal error LNK1120: 2 unresolved externals
[51/55] C:\Windows\system32\cmd.exe /C "cd . && D:\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=source\drivers\null\CMakeFiles\ze_null.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\link.exe  source\drivers\null\CMakeFiles\ze_null.dir\ze_null.cpp.obj source\drivers\null\CMakeFiles\ze_null.dir\ze_nullddi.cpp.obj source\drivers\null\CMakeFiles\ze_null.dir\zet_nullddi.cpp.obj source\drivers\null\CMakeFiles\ze_null.dir\zes_nullddi.cpp.obj  /out:bin\ze_null.dll /implib:lib\ze_null.lib /pdb:bin\ze_null.pdb /dll /version:1.20 /machine:x64 /DYNAMICBASE /guard:cf /CETCOMPAT /nologo    /debug /INCREMENTAL  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
[52/55] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1443~1.348\bin\Hostx64\x64\cl.exe   /TP -DL0_LOADER_VERSION=\"1\" -DL0_VALIDATION_LAYER_SUPPORTED_VERSION=\"1\" -DLOADER_VERSION_MAJOR=1 -DLOADER_VERSION_MINOR=20 -DLOADER_VERSION_PATCH=5 -DLOADER_VERSION_SHA="\" - No git SHA found, compiled outside git folder.\"" -Dze_tracing_layer_EXPORTS -ID:\b\level-zero\src\v1.20.5-0fe9d0b839.clean\include -ID:\b\level-zero\src\v1.20.5-0fe9d0b839.clean\source\wrapper\include -ID:\b\level-zero\src\v1.20.5-0fe9d0b839.clean -ID:\b\level-zero\src\v1.20.5-0fe9d0b839.clean\source\inc -ID:\b\level-zero\src\v1.20.5-0fe9d0b839.clean\source\layers\tracing /nologo /DWIN32 /D_WINDOWS /utf-8 /GR /EHsc /MP  /guard:cf /DYNAMICBASE /Qspectre /WX /W3 /MP /EHsc /MTd /Z7 /Ob0 /Od /RTC1  -std:c++14 /showIncludes /Fosource\layers\tracing\CMakeFiles\ze_tracing_layer.dir\ze_trcddi.cpp.obj /Fdsource\layers\tracing\CMakeFiles\ze_tracing_layer.dir\ /FS -c D:\b\level-zero\src\v1.20.5-0fe9d0b839.clean\source\layers\tracing\ze_trcddi.cpp
ninja: build stopped: subcommand failed.
```

--- a/ports/level-zero/patches/spdlog_include.patch
+++ b/ports/level-zero/patches/spdlog_include.patch
@@ -1,0 +1,13 @@
+diff --git a/source/utils/CMakeLists.txt b/source/utils/CMakeLists.txt
+index b77f264..de3f74d 100644
+--- a/source/utils/CMakeLists.txt
++++ b/source/utils/CMakeLists.txt
+@@ -5,7 +5,7 @@ set(logging_files logging.h logging.cpp)
+ add_library(level_zero_utils STATIC ${logging_files})
+ 
+ if(SYSTEM_SPDLOG)
+-       target_link_libraries(level_zero_utils PUBLIC spdlog::spdlog)
++       target_link_libraries(level_zero_utils PUBLIC spdlog::spdlog_header_only)
+ else()
+        target_include_directories(level_zero_utils PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/third_party/spdlog_headers>)
+ endif()

--- a/ports/level-zero/portfile.cmake
+++ b/ports/level-zero/portfile.cmake
@@ -4,8 +4,10 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO oneapi-src/level-zero
     REF "v${VERSION}"
-    SHA512 86bd21b034aaa4e0ce8f554a6563f41e622fcf39895c012dcc79d472e35f878dc759b103e9d2b7daac71e4943344a7afddd1f3c7ab889949b09b4cf8e0947589
+    SHA512 de83e691a8ef4f28fcd86aa919f8aae493b84c6b644b04efcc46cec405a3a0f3eab519ab78fda26d65161e8c6776723a090a412f24fa0564679d02258643f9d0
     HEAD_REF master
+    PATCHES
+        patches/spdlog_include.patch
 )
 
 vcpkg_cmake_configure(

--- a/ports/level-zero/vcpkg.json
+++ b/ports/level-zero/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "level-zero",
-  "version": "1.20.2",
+  "version": "1.20.5",
   "description": "oneAPI Level Zero Specification Headers and Loader.",
   "homepage": "https://github.com/oneapi-src/level-zero",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4321,7 +4321,7 @@
       "port-version": 0
     },
     "level-zero": {
-      "baseline": "1.20.2",
+      "baseline": "1.20.5",
       "port-version": 0
     },
     "leveldb": {

--- a/versions/l-/level-zero.json
+++ b/versions/l-/level-zero.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9979ff06cb5187169a02122503ddb9080c31fe11",
+      "version": "1.20.5",
+      "port-version": 0
+    },
+    {
       "git-tree": "a590b375a98696f4eba3c57c66a11141c3c7a77e",
       "version": "1.20.2",
       "port-version": 0

